grpc/src/main/java/com/linecorp/armeria/internal/grpc/ArmeriaMessageDeframer.java

```diff
@@ -257,18 +257,33 @@ public void deframe(HttpData data, boolean endOfStream) {
         deliver();
     }
 
+    /** Close when any messages currently queued have been requested and delivered. */
```

Comment:
nit: Close -> Closes?

Reply:
Thanks updated. I was a bit unsure since I just copied this line from upstream, but it makes sense for us to make tweaks since it's quite diverged at this point.